apps/payments/next/.env.development

```diff
@@ -75,6 +75,12 @@ STATS_D_CONFIG__HOST=
 STATS_D_CONFIG__PORT=
 STATS_D_CONFIG__PREFIX=
 
+# Glean Config
+GLEAN_CONFIG__APPLICATION_ID=
+# GLEAN_CONFIG__VERSION= # Set in next.config.js
```

Comment:
Should this not be written as follows instead?:
```
# Set in next.config.js
GLEAN_CONFIG__VERSION=
```

Reply:
The intention here is to show the developer that this config value exists, but that it's not set in any `.env` file, but instead is set in `next.config.js`. Leaving the comment on the same line as the config var, IMO more clearly indicates that the comment only applies to this line.